### PR TITLE
Added custom NTP servers

### LINKED ISSUE #113: If the clocks are not sychronized, OCS doesn't start
While deploying OCS, the ocs-operator pod stuck at 0/1 running. Digging a bit, the mon pods complain because clock not synchronized:

```
cluster 2019-09-10 13:35:31.525413 mon.a (mon.0) 33 : cluster [WRN] Health check failed: clock skew detected on mon.b (MON_CLOCK_SKEW)   
```

The install-scripts should define an NTP server and ensure the hosts are synchronized to it.

### PR DESCRIPTION
**Pull request by e-minguez**
This is basically a copy & paste from dev-scripts.
Fix #113

### CODE REVIEW

**Comment by e-minguez** [comment]
This is basically a copy & paste from dev-scripts.
Fix #113

**Comment by sreichar** [comment]
I don't plan on merging this.

For the appliance the plan is to see if the external network provided an NTP source, if not, we provide the NTP source using the mellanox switch, using the dnsmasq used to provide the storage IP addresses.

At least in theory, just getting the switch running now.

**Comment by e-minguez** [comment]
I moved it to the post-install script so it will be only applied if the variable NTP_SERVERS exist, otherwise it won't do anything.

File: OpenShift/99_post_install.sh [hunk 1]
```diff
@@ -64,5 +64,19 @@ apply_mc(){
   done
 }
 
+function create_ntp_config(){
+  if [ "$NTP_SERVERS" ]; then
+    cp assets/post-install/99_worker-chronyd-custom.yaml{.optional,}
```
Comment (sreichar) [inline OpenShift/99_post_install.sh]: Should you bracket the variable?

File: OpenShift/99_post_install.sh [hunk 2]
```diff
@@ -64,5 +64,19 @@ apply_mc(){
...
+    cp assets/post-install/99_master-chronyd-custom.yaml{.optional,}
+    NTPFILECONTENT=$(cat assets/files/etc/chrony.conf)
+    for ntp in $(echo $NTP_SERVERS | tr ";" "\n"); do
```
Comment (sreichar) [inline OpenShift/99_post_install.sh]: Again probably best to bracket the variable.

File: OpenShift/99_post_install.sh (hunk 1 above)
Comment (e-minguez) [reply OpenShift/99_post_install.sh]: Fixed, thanks!

**Comment by sreichar** [comment]
[test]

**Comment by sreichar** [comment]
@e-minguez with CI not working right now, did you test this?

**Comment by johnbieren** [comment]
[test] going to take about 5 hours until results are posted (2.5 hours for the other one I just requeued, then 2.5 hours for this one), but hopefully results by morning if it isn't merged by then

File: OpenShift/99_post_install.sh [hunk 3]
```diff
@@ -47,7 +47,7 @@ apply_mc(){
 
   # Apply machine configs
   for node_type in master worker; do
-    if test $(find "${POSTINSTALL_ASSETS_DIR}" -iname "*-${node_type}.yaml" -type f); then
+    if $(ls ${POSTINSTALL_ASSETS_DIR}/*-${node_type}.yaml >& /dev/null); then
       echo "Applying machine configs..."
       oc create -f ${POSTINSTALL_ASSETS_DIR}/*-${node_type}.yaml
```
[the remark this answers is not in the document]
Comment (e-minguez) [reply OpenShift/99_post_install.sh]: Thanks!

**Comment by e-minguez** [comment]
[test]

**Comment by e-minguez** [comment]
[test]

**Comment by johnbieren** [comment]
Previous CI run passed, job failed due to some java issue when saving the logs (not related). PR should be good to merge

**Comment by e-minguez** [comment]
ping @sreichar

**Pull request merged**